### PR TITLE
feature: add support for dcload, dcsource, alternator and DCDC

### PR DESCRIPTION
**Pull request by nmbath**
Updated to add additional paths to report information for dcdc, alternator, dcload and dcsource paths.

Has been running on my system for a few weeks, with no issues.

image

image

image

### CODE REVIEW

**Comment by mpvader** [comment]
Nice @nmbath 👍. one thing: note that those paths might still change. Probably not change by a lot, but usually we wait with things like this a bit until the source is in master / included in Venus. Especiially with the dcdc I’ve at least -still..- not thought it through.

**Comment by nmbath** [comment]
Understood, say enough to modify the paths once locked. Still need at add some of the further patch for alternator RPM, temperature and some further state information. So still some work to do to get all date available in SignalK.

File: venusToDeltas.js [hunk 1]
```diff
@@ -654,7 +654,15 @@ module.exports = function (app, options, handleMessage) {
       type = 'vecan'
     } else if ( msg.senderName.startsWith('com.victronenergy.hub4')) {
       type = 'hub4'
-    } else {
+	} else if ( msg.senderName.startsWith('com.victronenergy.dcsource')) {
```
Comment (sbender9) [inline venusToDeltas.js:657]: looks like there are some issues with spaces/tabs here?

**Comment by sbender9** [comment]
@mpvader @nmbath Seems like dcdc is not relevant now since the Orion XS is coming in as alternator.

According to this https://github.com/victronenergy/venus/wiki/dbus#alternator-and-dcsource , it looks like there is no /Mode for Wakespeed, but there is for the Orion XS. Is that correct?

Thinking this could start being an issue that I can't tell the difference between an alternator and an XS?

**Pull request merged**